client/app/stories/index.jsx

```diff
@@ -78,15 +85,63 @@ storiesOf('Input', module)
     <Input dark large label="Hello" placeholder="Placeholder" />
   ));
 
+// eslint-disable-next-line react/prefer-stateless-function
+class I18nWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      locale: defaultLocale,
+    };
+  }
+
+  render() {
+    const TitleComponent = injectIntl(({ intl }) => (
+      <div>
+        <p>{intl.formatMessage(defaultMessages.appDescription)}</p>
```

Comment:
I don't think we need this paragraph tag since it's already in a div.

Reply:
Yup, works for me